pom.xml

```diff
@@ -26,6 +26,9 @@
   <node.version>v14.17.0</node.version>
   <yarn.version>v1.22.13</yarn.version>
 
+  <postgres.image>docker.io/postgres</postgres.image>
+  <postgres.version>13-bullseye</postgres.version>
```

Comment:
Any specific reason to switch to `-bulleye`?

Reply:
https://hub.docker.com/_/postgres

No good anchor link on the page so Ctrl+F for "Arbitrary --user Notes"

tl;dr the default image is based on Alpine Linux, `-bullseye` is Debian. The Debian version of the image has some extra magic built in that lets it run under your host system UID/GID, so mounting the init script as a shared volume owned by your user works properly.

Reply:
Ahh makes sense! Thanks :D